### PR TITLE
feat(core): Added old wallet menu

### PR DESCRIPTION
**Pull request by amanCypherock**
Adds old wallet menu navigation only.

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-4839 Task controller merge - View wallet seed](https://app.clickup.com/t/9002019994/PRF-4839)

File: src/menu/wallet_menu.c [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  return;
+}
+
+/*****************************************************************************
+ * GLOBAL FUNCTIONS
+ *****************************************************************************/
+const flow_step_t *wallet_menu_get_step(uint8_t selection_index) {
+  if (0 == selection_index || MAX_WALLETS_ALLOWED < selection_index) {
+    return NULL;
+  }
```
Comment (kartikCypherock) [inline src/menu/wallet_menu.c:197]: How will this be caught?

**Review by kartikCypherock — APPROVED** [review]

File: src/menu/wallet_menu.c (hunk 1 above)
Comment (amanCypherock) [reply src/menu/wallet_menu.c:197]: Agreed we don't want this case to occur at all.

Comment (amanCypherock) [reply src/menu/wallet_menu.c]: updated

**Pull request merged**